zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/DefaultExporterResourceProvider.java

```diff
@@ -80,7 +82,9 @@ public void init(final ExporterConfiguration configuration) {
                 templateDescriptorsMap.get(ListViewTemplate.class).getFullQualifiedName(), false),
             new VariableHandler(
                 templateDescriptorsMap.get(VariableTemplate.class).getFullQualifiedName(),
-                configuration.getIndex().getVariableSizeThreshold()));
+                configuration.getIndex().getVariableSizeThreshold()),
+            new ProcessHandler(
+                indexDescriptorsMap.get(ProcessIndex.class).getFullQualifiedName(), new XMLUtil()));
```

Comment:
🔧 you could have initialized it inside the `ProcessHandler` constructor, no need to pass it around. but this works too.

Reply:
True, left it in the provider in case any external configuration can be passed in the future in the XMLUtil, as a DI placeholder.